.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--- PR title format: [GH#<Issue ID>][BZ<BZ ID>][<Jira ID>]: <Short description of your PR> --->
+
+
+Version:
+<!--- What version of OpenShift does this apply to? --->
+
+
+Issue:
+<!--- Add a link to your Bugzilla, Jira, or GitHub issue. --->
+
+
+Docs Preview:
+<!--- Add the direct link to your content. --->
+
+Summary of changes
```

Comment:
Just wondering if this part is necessary? I'd normally think that the commit message would be sufficient in describing the changes.

Maybe if we do want to keep an area for people to add more info if required, maybe it could be something like "Additional information" or something so that it looks more optional?